BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.kt

```diff
@@ -3,24 +3,30 @@ package com.braintreepayments.api
 import android.content.Context
 import androidx.room.AutoMigration
 import androidx.room.Database
+import androidx.room.DeleteTable
 import androidx.room.Room
 import androidx.room.RoomDatabase
+import androidx.room.migration.AutoMigrationSpec
 
 // Ref: https://developer.android.com/training/data-storage/room/migrating-db-versions
 @Database(
-        version = 6,
-        entities = [AnalyticsEvent::class],
-        autoMigrations = [
-            AutoMigration(from = 1, to = 2),
-            AutoMigration(from = 2, to = 3),
-            AutoMigration(from = 3, to = 4),
-            AutoMigration(from = 4, to = 5),
-            AutoMigration(from = 5, to = 6),
-        ]
+    version = 7,
+    entities = [AnalyticsEventBlob::class],
+    autoMigrations = [
+        AutoMigration(from = 1, to = 2),
+        AutoMigration(from = 2, to = 3),
+        AutoMigration(from = 3, to = 4),
+        AutoMigration(from = 4, to = 5),
+        AutoMigration(from = 5, to = 6),
+        AutoMigration(from = 6, to = 7, spec = AnalyticsDatabase.DeleteAnalyticsEventTableAutoMigration::class)
```

Comment:
This should be the last migration we'll ever need to do ❤️

Reply:
🤞 yeah once we moved from BT analytics to FPTI the requirements changed. Since there could be new analytics fields in the future, it makes sense to store the analytics request body as a string in the DB.